src/BlockchainTestsFiller/bcUncleHeaderValiditiy/futureUncleTimestampDifficultyDrop2Filler.json

```diff
@@ -88,6 +77,7 @@
 		"expectExceptionEIP150" : "InvalidStateRoot",
 		"expectExceptionEIP158" : "InvalidStateRoot",
                 "expectExceptionByzantium" : "InvalidStateRoot",
+                "expectExceptionConstantinople" : "InvalidStateRoot",
 		"blocknumber" : "3",
 		"blockHeader" : {
 			"stateRoot" : "3e9110dccee270e36e254d8e3b67cb18c12a9a0aa24065e24372a85856195c70",
```

Comment:
I see. the trick here is that we change timestamp deliberately because uncle has relTimestamp change. 
you have to find what is the new stateRoot of a block with uncle and put that value in here. then there would be no expectException